torch/csrc/Module.cpp

```diff
@@ -454,7 +454,9 @@ PyObject *THPModule_setDeterministic(PyObject *_unused, PyObject *arg)
 {
   THPUtils_assert(PyBool_Check(arg), "set_deterministic expects a bool, "
           "but got %s", THPUtils_typename(arg));
+  HANDLE_TH_ERRORS
```

Comment:
nit: This usually enclose the whole body of the function (including arg checking and return statement).

Reply:
I'm also not sure why this would be added in this PR.

Reply:
removed